tests/src/test/java/alluxio/client/FileInStreamIntegrationTest.java

```diff
@@ -57,9 +58,12 @@
   @BeforeClass
   public static final void beforeClass() throws Exception {
     sFileSystem = sLocalAlluxioClusterResource.get().getClient();
-    sWriteBoth = CreateFileOptions.defaults().setWriteType(WriteType.CACHE_THROUGH);
-    sWriteAlluxio = CreateFileOptions.defaults().setWriteType(WriteType.MUST_CACHE);
-    sWriteUnderStore = CreateFileOptions.defaults().setWriteType(WriteType.THROUGH);
+    sWriteBoth = CreateFileOptions.defaults().setMode(Mode.createFullAccess())
```

Comment:
is this change required? how it works before this PR as I don't see real code change related in this PR?